packages/client/composables/useContext.ts

```diff
@@ -1,22 +1,16 @@
-import type { ComputedRef, WritableComputedRef } from 'vue'
+import type { ComputedRef } from 'vue'
 import { computed } from 'vue'
 import type { RouteLocationNormalizedLoaded } from 'vue-router'
 import type { SlidevContext } from '../modules/context'
 import { configs } from '../env'
 import { useNav } from './useNav'
-import { useNavClicks } from './useNavClicks'
 
 export function useContext(
   route: ComputedRef<RouteLocationNormalizedLoaded>,
-  clicks: WritableComputedRef<number>,
 ): SlidevContext {
   const nav = useNav(route)
-  const navClicks = useNavClicks(clicks, nav.currentRoute, nav.currentPage)
   return {
-    nav: {
-      ...nav,
-      ...navClicks,
```

Comment:
I feel we might still need this, that `clicks`, `clicksTotal`, `clicksContext` should resolved from injections (as we have primary clicks and static clicks now). Components calling this function should get the context to the state of that slide